src/main/scala/com/atomist/tree/content/text/microgrammar/Matcher.scala

```diff
@@ -23,7 +23,16 @@ trait Matcher {
     * @param inputState input state
     * @return match or failure to match
     */
-  def matchPrefix(inputState: InputState): Option[PatternMatch]
+  def matchPrefixInternal(inputState: InputState): Option[PatternMatch]
```

Comment:
yep. I wanted it to be, and I think I've changed everything to use matchPrefix already.